### PR TITLE
JP-3656: Removed unused DrizPars schemas and models

### LINKED ISSUE #304: Removed unused `DrizPars` stuff
xlinking: https://github.com/spacetelescope/jwst/issues/8558

https://github.com/search?q=repo%3Aspacetelescope%2Fstdatamodels+DRIZPAR&type=code

Some things that might be removable:
- [DrizParsModel](https://github.com/spacetelescope/stdatamodels/blob/main/src/stdatamodels/jwst/datamodels/drizpars.py)
- [drizpars.schema.yaml](https://github.com/spacetelescope/stdatamodels/blob/main/src/stdatamodels/jwst/datamodels/schemas/drizpars.schema.yaml)
- [R_DRZPAR](https://github.com/spacetelescope/stdatamodels/blob/0326fd4aa771c42a731eb0e5e0bd9bb2fc267789/src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml#L1831) in core schema

Targeting inclusion in jwst B11.1

### PR DESCRIPTION
**Pull request by emolter**
Resolves [JP-3656](https://jira.stsci.edu/browse/JP-3656)

Closes [JWST 8558](https://github.com/spacetelescope/jwst/issues/8558)
Closes #304 

This PR removes the unused DrizPars model type for JWST and the associated R_DRIZPAR keyword in the core schema, as well as associated imports and docs.

**Checklist**
- [x] added entry in `CHANGES.rst` (either in `Bug Fixes` or `Changes to API`)
- [x] updated relevant tests
- [x] updated relevant documentation
- [ ] updated relevant milestone(s)
- [ ] added relevant label(s)

### CODE REVIEW

**Comment by emolter** [comment]
Four failing tests here.  One matches the nightly test runs; the other three appear to be related to a necessary CRDS update.  Can someone please advise me on where those come from and how they should be fixed?  I would be happy to make a PR in CRDS (if that's indeed what's required), but I've never touched that code before and could use some guidance.

**Comment by braingram** [comment]
The `drizpars` ones are a result of the tests that this repository contains that check that "all" reference files from crds can be loaded. Those failures are expected since `drizpars` support is removed with this PR.

I would be fine with disabling the tests against the `drizpars` reference files. @tapastro any objection?

The other failure on main is unrelated. Presumably it's from a crds update.

**Comment by braingram** [comment]
If ignoring `drizpars` is acceptable, it can be skipped somewhere here:
https://github.com/spacetelescope/stdatamodels/blob/ee4756ffe9ff71c1fb7b067f8998949a88d6a703/src/stdatamodels/jwst/datamodels/tests/test_schema_against_crds.py#L194-L200
(similar to how the `pars-` files are skipped).

**Comment by braingram** [comment]
Would you run regtests with this PR? I'm assuming it will show many expected failures due to the missing `R_DRIZPAR`.

**Comment by emolter** [comment]
regression tests for JWST[ started here](https://plwishmaster.stsci.edu:8081/job/RT/job/JWST-Developers-Pull-Requests/1654/)

File: src/stdatamodels/jwst/datamodels/tests/test_schema_against_crds.py [hunk 1]
```diff
@@ -32,6 +32,9 @@ def flatten(xs):
                    'META.OBSERVATION.TIME',
                    ]
 
+# Reference types to ignore, e.g. because they have been deprecated
```
Comment (braingram) [inline src/stdatamodels/jwst/datamodels/tests/test_schema_against_crds.py:35]:
```suggestion
# Ignore pars files and deprecated files
```

**Review by braingram — APPROVED** [review]
Just one suggestion to the comment wording to clarify that `pars` files aren't deprecated.

Thanks!

**Comment by braingram** [comment]
I'll leave merging up to you (let me know if you don't have permission) to schedule the regtest okifying. Thanks!

**Comment by emolter** [comment]
@tapastro are the JWST regression tests in a good enough state for us to merge this and for me to ok-ify as I see fit afterward?

**Review by tapastro — APPROVED** [review]
LGTM, thank you for your patience 🙂

**Comment by emolter** [comment]
regression tests to okify are [started here](https://github.com/spacetelescope/RegressionTests/actions/runs/10780450832)

**Pull request merged**